README.md

```diff
@@ -79,6 +80,33 @@ If the source file does not follow the pattern `create[custom element]Element`,
 dojo build webpack --element=src/path/to/element.ts --elementPrefix=the-special
 ```
 
+### Feature optimization
+
+This command supports the ability to optimize code based on statically asserted features.  The tool can search the source code for modules that attempt to detect features using a [`@dojo/has`](https://github.com/dojo/has) type of API.  By supplying a feature set (or sets) on the command line, the build will be optimized to optimize code branches, making the code smaller and slightly more efficient.  This allows targeting of particular platforms.  The tool will not optimize code, when specifying multiple feature sets, they do not align.  It will instead continue to leave that feature to be detected at run-time.
+
+From the command line, the feature sets are provided to the `-f` or `--feature` argument.  The available feature sets are aligned to platforms.  The currently available feature sets are:
+
+|Feature Set|Description|
```

Comment:
People may not be impressed to learn that we have what they might think of a browser sniff testing rather than feature testing, so perhaps it's worth explaining a bit more how these feature test lists were created?